spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-configmap-event-reload/src/test/resources/three/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
```

Comment:
there are 3 deployments, all activate different profiles, as such different yaml files are loaded, and as suc different things are tested.